nngt/plot/plt_properties.py

```diff
@@ -133,11 +147,19 @@ def degree_distribution(network, deg_type="total", nodes=None,
 
     axis.set_xlabel("Degree")
     axis.set_ylabel("Node count")
+    # title_start = (deg_type[0].upper() + deg_type[1:] + '-d'
+    #                if isinstance(deg_type, str) else 'D')
     title_start = (deg_type[0].upper() + deg_type[1:] + '-d'
-                   if isinstance(deg_type, str) else 'D')
-    axis.set_title(
-        "{}egree distribution for {}".format(title_start, network.name), x=0.,
-        y=1.05, loc='left')
+                   if isinstance(deg_type, str) else s_type[0].upper() + s_type[1:] + " D")
+
+    if title is not "": # if an optional title is provided at the function call
+        axis.set_title(
+            "{0} : {1}egree distribution for {2}".format(title,title_start, network.name), x=0.,
```

Comment:
Remove whitespace before column, remove ``if else`` blocks by just adding ``: `` to ``title`` if it is not empty:

```python
title += ": " if title else ""
axis.set_title("{}{}egree distribution for {}.".format(title,title_start, network.name), x=0., y=1.05, loc='left')
```

do not use ints for string formatting, either use keywords and keywords arguments, or leave braces empty.